src/components/ui/ListItem.svelte

```diff
@@ -1,10 +1,16 @@
 <script lang="ts">
   import { createEventDispatcher } from 'svelte';
+  import type { Action } from 'svelte/action';
 
   export { className as class };
   export { styleName as style };
   export let draggable: boolean = false;
 
+  export let tooltip: Action<HTMLElement, any> | undefined = undefined;
+  export let tooltipContent: any = undefined;
+  export let tooltipPlacement: string = 'top';
```

Comment:
Since it doesn't look like you're specifying `tooltipPlacement` in the component using this, you probably don't need to make this a whole prop for `ListItem`.